test_unstructured/partition/test_auto.py

```diff
@@ -200,14 +201,6 @@ def test_auto_partition_email_from_file():
     assert elements == EXPECTED_EMAIL_OUTPUT
 
 
-def test_auto_partition_eml_add_signature_to_metadata():
-    elements = partition(example_doc_path("eml/signed-doc.p7s"))
-
-    assert len(elements) == 1
-    assert elements[0].text == "This is a test"
-    assert elements[0].metadata.signature == "<SIGNATURE>\n"
```

Comment:
Is `metadata.signature` becoming not supported with these changes?

Reply:
Right. Matt and I decided that should go as it is large-ish, would be repeated on each element, and is of no direct use without a public key.

We did talk about possibly adding `.metadata.is_signed` as a bool field but that goes on the backlog.